buildroot/package/rpi-fbcp/rpi-fbcp.mk

```diff
@@ -0,0 +1,12 @@
+################################################################################
```

Comment:
Nice job on this package, is this patch submitted upstream?

I searched the buildroot mailing list, and in the process, found that somebody else submitted an rpi-fbcp package. A reviewer left feedback on it, but it was abandoned.

Reply:
It's not submitted upstream as I haven't been able to get it to test it yet! So far all my builds seem to just skip this recipe even though it's been added to packages and included in my config. Can you see what I'm doing wrong?